docs/config/connpass.json

```
@@ -209,6 +209,10 @@
       "id": "kashiwarb",
       "name": "Kashiwa.rb",
       "series_id": 15086
+    },{
```

I made it the same as the other rows. (I prefer this)

```suggestion
    },
    {
```

---

@sue445 

Sorry for overlooking my careless mistake 🙏